src/Services/HubspotCommerce.php

```diff
@@ -0,0 +1,140 @@
+<?php
+
+namespace Drupal\hubspot_commerce\Services;
+
+use Drupal\Core\Entity\EntityInterface;
+use Drupal\hubspot_api\Manager;
+
+use Drupal\Core\Entity\EntityTypeManagerInterface;
+use Drupal\Core\Logger\LoggerChannelFactoryInterface;
+use Exception;
+
+/**
+ * The HubspotCommerce service class.
+ *
+ * Contains functions to sync data with the Hubspot API.
+ *
+ * @package Drupal\hubspot_commerce
+ */
+class HubspotCommerce {
```

Comment:
**CHANGE REQUEST**

`implements SyncToInterface` that defines the `sync` function.